src/main/java/org/craftercms/studio/impl/v1/service/content/ContentServiceImpl.java

```diff
@@ -2586,6 +2603,18 @@ public boolean pullFromRemote(String siteId, String remoteName, String remoteBra
         return toRet;
     }
 
+    //TODO: Move this to a fixed always-on site policy when implemented
+    protected void validateContent(String path, String filename) throws ServiceLayerException {
+        if (filename.length() >= 255) {
```

Comment:
Please put the numbers 255, 1024 as configuration items in studio-config.yaml. This will help us configure the DB and Studio more easily if we have to, and avoids the magic number here. I expect this configuration to be in RAM and not pulled from the configuration file everytime.

Reply:
I can make it configurable for the service, but not for DB because those are in the create schema script it's not possible to inject them there

Reply:
Yes, understood. Go ahead and make configurable, please. Also, make a note in the configuration file that DB changes are also required. Please don't propagate this to studio-config-override, only keep studio-config.yaml.